src/Tooltip.js

```diff
@@ -169,16 +169,31 @@ class Tooltip extends React.Component<Props, State> {
   }
 
   getElementPosition = (event) => {
-    this.renderedElement && this.renderedElement.measure(
-      (frameOffsetX, frameOffsetY, width, height, pageOffsetX, pageOffsetY) => {
-        this.setState({
-          xOffset: pageOffsetX,
-          yOffset: pageOffsetY,
-          elementWidth: width,
-          elementHeight: height,
-        });
-      }
-    );
+    const {noStatusBar} = this.props
+    if( noStatusBar ){
```

Comment:
Strange prettier didn't format this?, did make the change locally? could you run prettier on this?